services/src/main/java/org/fao/geonet/api/links/LinksApi.java

```diff
@@ -176,8 +212,10 @@ public ResponseEntity analyzeRecordLinks(
         @ApiIgnore
             HttpServletRequest request
     ) throws IOException, JDOMException {
+        MAnalyseProcess registredMAnalyseProcess = getRegistredMAnalyseProcess();
+
         if (removeFirst) {
```

Comment:
@cmangeat, can you indicate what would be the result of issuing this API method while already been executed while a previous request? Seem to me that removes previous execution, but not very clear really.

Reply:
Yes it does. Maybe we are a little schizophrene at this point, wanting both:
- an automatic batch process, deleting all and rebuilding the whole, "black box",
- ability to trigger targeted processes, related to one metadata or one link, runnable concurrently, adapted to object oriented gui.